internal/store/projects.go

```diff
@@ -113,3 +117,17 @@ func (s *Store) DeleteProject(projectID string) error {
 		return fmt.Errorf("unsupported service: %s", s.service)
 	}
 }
+
+// Users lists all IAM users in a project
+func (s *Store) Users(projectID string, opts *atlas.ListOptions) (interface{}, error) {
```

Comment:
[nit] since we're going to have organization users and any user commands this may be better named `ProjectUsers`